docs/running_on_mainnet.md

```diff
@@ -1 +1,121 @@
+
+
+## Steps
+
+1. Clone and Install Bittensor and the Bittensor Subnet Template.
+This clones and installs the template if you dont already have it (if you do, skip this step)
+```bash
+cd .. # back out of the subtensor repo
+git clone https://github.com/opentensor/bittensor-subnet-template.git # Clone the bittensor-subnet-template repo
+cd bittensor-subnet-template # Enter the bittensor-subnet-template repo
+python -m pip install -e . # Install the bittensor-subnet-template package
+```
+
+2. Create wallets for your subnet owner, for your validator and for your miner.
+This creates local coldkey and hotkey pairs for your 3 identities. The owner will create and control the subnet and must have at least 100 TAO on it before it can run step 3. The validator and miner will run the respective validator/miner scripts and be registered to the subnetwork created by the owner.
+```bash
+# Create a coldkey for your owner wallet.
+btcli wallet new_coldkey --wallet.name owner
+
+# Create a coldkey and hotkey for your miner wallet.
+btcli wallet new_coldkey --wallet.name miner
+btcli wallet new_hotkey --wallet.name miner --wallet.hotkey default
+
+# Create a coldkey and hotkey for your validator wallet.
+btcli wallet new_coldkey --wallet.name validator
+btcli wallet new_hotkey --wallet.name validator --wallet.hotkey default
+```
+
+3. Getting the price of subnetwork creation
+Creating subnetworks on mainnet is competitive and the cost it determined by the rate at which new network are being registered onto the chain. By default you must have at least 100 TAO on your owner wallet to create a subnetwork. However the exact amount will fluctuate based on demand. The code below shows how to get the current price of creating a subnetwork.
+```bash
+btcli subnet lock_cost --subtensor.network finney
+>> Subnet lock cost: τ100.000000000
+```
+
+
+4. Purchasing a slot
+Using your TAO balance, you can register your subnet to the chain. This will create a new subnet on the chain and give you the owner permissions to it. The code below shows how to purchase a slot. 
+*Note: Slots cost TAO to lock, you will get this TAO back when the subnet is disolved.*
+```bash
+# Run the register subnetwork command on the locally running chain.
+btcli subnet create --subtensor.network finney 
+# Enter the owner wallet name which gives permissions to the coldkey to later define running hyper parameters.
+>> Enter wallet name (default): owner # Enter your owner wallet name
+>> Enter password to unlock key: # Enter your wallet password.
+>> Register subnet? [y/n]: <y/n> # Select yes (y)
+>> ⠇ 📡 Registering subnet...
+✅ Registered subnetwork with netuid: 1 # Your subnet netuid will show here, save this for later.
+```
+
+5. Register your validator and miner keys to the networks.
```

Comment:
Neither of these are **required** steps AFAIK. Subnet owner can now sit back with a piña colada and watch the games begin

Reply:
Making it (optional) but will leave for reference. Ideally subnet owners will be leading the way by showing examples of running a miner/validator but agree it's not strictly necessary.